pkg/server/version.go

```diff
@@ -17,32 +17,29 @@ limitations under the License.
 package server
 
 import (
+	"fmt"
+
 	"golang.org/x/net/context"
 	"k8s.io/kubernetes/pkg/kubelet/apis/cri/v1alpha1/runtime"
-
-	"github.com/containerd/cri-containerd/pkg/version"
 )
 
 const (
-	// For now, containerd and runc are bundled with cri-containerd, cri-containerd
-	// version is more important to us.
-	// TODO(random-liu): Figure out how to package cri-containerd and containerd,
-	// and how to version it. We still prefer calling the container runtime "containerd",
-	// but we care both the cri-containerd version and containerd version.
-	containerName        = "cri-containerd"
-	containerdAPIVersion = "0.0.0"
+	containerName = "containerd"
 	// kubeAPIVersion is the api version of kubernetes.
 	kubeAPIVersion = "0.1.0"
 )
 
 // Version returns the runtime name, runtime version and runtime API version.
-// TODO(random-liu): Return containerd version since we are going to merge 2 daemons.
 func (c *criContainerdService) Version(ctx context.Context, r *runtime.VersionRequest) (*runtime.VersionResponse, error) {
+	resp, err := c.client.Version(ctx)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get containerd version: %v", err)
+	}
 	return &runtime.VersionResponse{
 		Version:        kubeAPIVersion,
 		RuntimeName:    containerName,
-		RuntimeVersion: version.CRIContainerdVersion,
```

Comment:
see..
https://github.com/containerd/cri-containerd/blob/master/Makefile#L31

Note: this is the only use of pkg/version.. so if we don't keep using it we'll want to update the makefile and rm pkg/version/version.go

Reply:
@mikebrow For plugin mode, we won't have our own version, containerd will take care of that.

But I agree, we should punt this change until containerd starts vendoring cri plugin.

Reply:
we may not have our own version via reporting but as long as we have our own github branch we'll at least have a commit version if no longer adding version tags to our release commits..

Reply:
and because we'll have a vendor'd in version tag/commit.. we may want to figure out how to report that